### PR TITLE
Rewrite the DOAP

### LINKED ISSUE #82: What to do with the DOAP file?
> DOAP (Description of a Project) is an RDF Schema and XML vocabulary to describe software projects, in particular free and open source software.

https://en.wikipedia.org/wiki/DOAP

---

It doesn't seem to be commonly used outside of GNOME, where every single project, library and app, has a `${name}.doap` file.

One thing that uses it is GNOME Builder. There, Glimpse shows as « GIMP », because that's how the project is named in the DOAP file.

I see a couple of possibilities:

* modify the DOAP file to have it contain correct information, and rename it to `glimpse.doap`;
* remove it entirely, considering how rare it is outside of GNOME.

Both options are fine.

In particular, when it can't find a `*.doap` file in a project, GNOME Builder will fallback on some other mechanisms to find the project name. (unverified, but most likely the `*.desktop` file, `configure.ac` or something like that) With #79 and #81 merged and the DOAP removed, Builder shows the project named as « Glimpse ».

The status quo is wrong and confusing though. Which option do people prefer?

### PR DESCRIPTION
**Pull request by bochecha**
Not only do we need to rename the DOAP file, we also need to rewrite its
content so they are adapted to Glimpse.

Fixes #82

### CODE REVIEW

File: glimpse.doap [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    </foaf:Person>
+  </maintainer>
+  <maintainer>
+    <foaf:Person>
+      <foaf:name>Chris</foaf:name>
+      <gnome:userid>BrainBlasted</gnome:userid>
+    </foaf:Person>
+  </maintainer>
+
+<!-- TODO: generate MAINTAINERS from this file -->
```
Comment (igalic) [inline glimpse.doap]: not sure all these people have a gnome user id, or, what that exactly is

Comment (TrechNex) [reply glimpse.doap]: Yep, can confirm I don't have one. Also there was an earlier change where we chose not to identify as a GNOME application, so it seems a bit weird that we're using their namespace for this.

Comment (bochecha) [reply glimpse.doap]: This is not a "GNOME user id".

In this context, "gnome" is an XML namespace. That is, this tag is the `<userid>` tag, which is defined in the "gnome" namespace, as declared in the `<Project>` tag:

```
xmlns:gnome="http://api.gnome.org/doap-extensions#"
```

(of course that URL doesn't work any more… I'll go and ask the GNOME sysadmins)

Comment (bochecha) [reply glimpse.doap]: It seems the `foaf` namespace now has a `nick` property: http://xmlns.com/foaf/spec/#term_nick

Would you prefer using that?

Comment (TrechNex) [reply glimpse.doap]: Looks fine to me :)

File: glimpse.doap [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  <category rdf:resource="http://api.gnome.org/doap-extensions#apps" />
+
+  <programming-language>C</programming-language>
+  <programming-language>Python</programming-language>
+  <programming-language>Scheme</programming-language>
+
+  <maintainer>
+    <foaf:Person>
+      <foaf:name>Bobby Moss</foaf:name>
+      <gnome:userid>TrechNexTrechNex</gnome:userid>
```
Comment (TrechNex) [inline glimpse.doap]: This should just be "TrechNex"

**Pull request merged**